src/components/Col/Col.js

```diff
@@ -20,7 +20,7 @@ import isPropValid from '@emotion/is-prop-valid';
 
 import { getSpanStyles, getSkipStyles, getBreakPointStyles } from './utils';
 
-const baseStyles = ({ theme, skip = '0', span = '0' }) => css`
+const baseStyles = ({ theme, span, skip }) => css`
```

Comment:
Was this the inferred type?

Reply:
Yes, that's what TypeScript inferred the types from.